.eslintrc.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
   plugins: ['react', , '@typescript-eslint'],
   rules: {
     'require-jsdoc': 0,
+    'react/display-name': 'off',
```

Comment:
display-name arguably should be kept on.. 🤔

Reply:
probably should, but I do find it annoying